src/main/scala/com/crealytics/spark/excel/v2/ExcelHelper.scala

```diff
@@ -216,6 +216,10 @@ class ExcelHelper private (options: ExcelOptions) {
       if (options.columnNameOfRowNumber.isDefined) Vector[String](options.columnNameOfRowNumber.get)
       else Vector.empty[String]
 
+    val rowIsHiddenColumn =
+      if (options.columnNameOfRowIsHidden.isDefined) Vector[String](options.columnNameOfRowIsHidden.get)
+      else Vector.empty[String]
```

Comment:
```suggestion
    val rowIsHiddenColumn = options.columnNameOfRowIsHidden.toSeq
```

Reply:
I incorporated this suggestion. It caused a type error with the returned value below, but I changed it to concatenate `rowIsHiddenColumn.toVector` and it seems to compile now. I'm not sure what the difference is between Sequence and Vector, but it seems simple enough to convert from one to the other.